sherpa-onnx/csrc/online-transducer-decoder.h

```diff
@@ -76,6 +78,27 @@ class OnlineTransducerDecoder {
   virtual void Decode(Ort::Value encoder_out,
                       std::vector<OnlineTransducerDecoderResult> *result) = 0;
 
+  /** Run transducer beam search given the output from the encoder model.
+   *
+   * Note: Currently this interface is for contextual-biasing feature which
+   *       needs a ContextGraph owning by the OnlineStream.
+   *
+   * @param encoder_out A 3-D tensor of shape (N, T, joiner_dim)
+   * @param ss  A list of OnlineStreams.
+   * @param num_streams  The number of OnlineStreams, should be equal to N.
```

Comment:
Do we really need this argument if it is always equal to N.

Also, when ss is nullptr, you set num_streams to 0, which is not equal to N.

Reply:
I think we can remove it.